tequilapi/endpoints/identities.go

```diff
@@ -195,6 +209,59 @@ func (endpoint *identitiesAPI) Unlock(resp http.ResponseWriter, request *http.Re
 	resp.WriteHeader(http.StatusAccepted)
 }
 
+// swagger:operation PUT /identities/{id}/payout Identity createPayoutInfo
+// ---
+// summary: Registers payout address
+// description: Registers payout address for identity
+// parameters:
+// - name: id
+//   in: path
+//   description: Identity stored in keystore
+//   type: string
+//   required: true
+// - in: body
+//   name: body
+//   description: Parameter in body (eth_address) is required
+//   schema:
+//     $ref: "#/definitions/PayoutInfoDTO"
+// responses:
+//   202:
+//     description: Identity registered
+//   400:
+//     description: Bad request
+//     schema:
+//       "$ref": "#/definitions/ErrorMessageDTO"
+//   500:
+//     description: Internal server error
+//     schema:
+//       "$ref": "#/definitions/ErrorMessageDTO"
+//   501:
```

Comment:
Please align documentation with code here:
- there is no error code 501
- 422 is missing for fields validation
- maybe smtg else?

Reply:
aligned